modules/admin_manual/pages/configuration/server/occ_commands/core_commands/_file_commands.adoc

```diff
@@ -120,6 +122,61 @@ For example, if the user's id was "john" and the `--path` value was "tree/apple"
 
 `files:cleanup` tidies up the server's file cache by deleting all file entries that have no matching entries in the storage table.
 
+== The files:remove-storage command
+
+When a mount point gets removed, the storage ID and file cache related info stay in the database. Over time, this can consume a considerable amount of database space and can slow down database queries. With the `files:remove-storage` command, these orphans can be removed. As a rule of thumb, though not essential, this command should be run in single user mode to avoid any conflicting access.
+
+[source,bash,subs="attributes+"]
+----
+{occ-command-example-prefix} files:remove-storage --help
+ Usage:
+   files:remove-storage [options] [--] [<storage-id>]...
+----
+
+=== Arguments
+
+[width="100%",cols="23%,70%",]
+|===
+| `storage-id` | The numeric ID of the storage.
+|===
+
+=== Options
+
+[width="100%",cols="25%,70%",]
+|===
+| `--chunk-size=[value]`   | The number of rows that will be deleted at the same time. Defaults to 1000.
+| `--show-candidates`      | Show possible candidates for obsolete storages. This query can take a while.
+|===
+
+[IMPORTANT]
+====
+This command should be handled with absolute care because *any* storage ID and the correlated file cache can be removed. For security reasons, the following advices are given:
```

Comment:
```suggestion
This command should be handled with absolute care because *any* storage ID and the correlated file cache can be removed. For safety reasons, the following advices are given:
```